RecoTracker/TkNavigation/test/NavigationSchoolAnalyzer.cc

```diff
@@ -75,36 +75,26 @@ void NavigationSchoolAnalyzer::print(std::ostream& os,const DetLayer* dl){
   unsigned int LorW=0;
   unsigned int side=0;
 
+  if(GeomDetEnumerators::isTracker(dl->subDetector())) {
+    LorW = tTopo->layer(tag->geographicalId());
+    side = tTopo->side(tag->geographicalId());
+  }
+  else {
   switch (dl->subDetector()){
```

Comment:
It looks like you accidentally deleted the entry for drift tubes:

 case GeomDetEnumerators::DT :
    LorW = DTChamberId(tag->geographicalId().rawId()).station(); break;

Aren't these lines needed?